lib/type/string.ex

```diff
@@ -36,6 +37,37 @@ defmodule Litmus.Type.String do
     end
   end
 
+  @spec convert(t, binary, map) :: {:ok, map}
+  defp convert(%__MODULE__{}, field, params) do
+    if Map.has_key?(params, field) and
+         !(is_binary(params[field]) or is_number(params[field]) or is_boolean(params[field])) do
+      {:error, "#{field} must be string"}
+    else
+      if is_binary(params[field]) do
+        {:ok, params}
+      else
+        modified_value = Kernel.inspect(params[field])
```

Comment:
`Kernel.inspect` is for inspect values for debugging and other non-user facing things. We should use `to_string/1`.